Doc/library/asyncio-eventloop.rst

```diff
@@ -186,19 +186,25 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
+   After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   The *timeout* parameter specifies the duration in seconds (as a :class:`float`)
+   this method gives the executor to finish joining.
+   With the default, ``None``, the
+   method allows the executor an unlimited amount of time.
 
-   If the timeout duration is reached, a warning is emitted and executor is
-   terminated without waiting for its threads to finish joining.
+   If the *timeout* is reached, this method emits a :exc:`RuntimeWarning` and
+   terminates the default executor without waiting for its threads to finish
+   joining.
```

Comment:
```suggestion
   If the *timeout* is reached, a :exc:`RuntimeWarning` is emitted
   and the default executor is terminated
   without waiting for its threads to finish joining.
```

Like before, reading and thinking over this again, passive voice is generally idiomatic and keeping with the Diataxis principles in API reference documentation in the case of referring to the construct that is explicitly stated as being described, and avoids implying implementation details that may or may not be entirely correct (e.g. it implies the method directly emits the `RuntimeWarning`, when it may in fact be emitted deeper in the execution stack).

Reply:
I was wondering about the implied implementation detail.  agree with leave it out.